pkg/operator/endpoints/metrics.go

```diff
@@ -0,0 +1,449 @@
+/*
+Copyright 2019 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endpoints
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+
+	"github.com/cortexlabs/cortex/pkg/operator/api/context"
+	"github.com/cortexlabs/cortex/pkg/operator/api/schema"
+	"github.com/cortexlabs/cortex/pkg/operator/api/userconfig"
+	"github.com/cortexlabs/cortex/pkg/operator/config"
+	"github.com/cortexlabs/cortex/pkg/operator/workloads"
+)
+
+const (
+	maxClasses        = 50
+	aggregationPeriod = int64(60 * 60) // 1 hour
+)
+
+func GetMetrics(w http.ResponseWriter, r *http.Request) {
+	appName, err := getRequiredQueryParam("appName", r)
+	if err != nil {
+		RespondError(w, err)
+		return
+	}
+
+	ctx := workloads.CurrentContext(appName)
+	err = workloads.PopulateWorkloadIDs(ctx)
```

Comment:
I don't think this is necessary, since I think workload IDs are populated before saving the ctx on `cx deploy`